app/components/Cards/PricingCard.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+import classes from "./PricingCard.module.css";
+import checkpoint from "../../../public/Icons/checkpoint_icon.svg";
+import Image from "next/image";
+
+const PricingCard = (props: any) => {
+  return (
+    <div
+      className={`${classes["pricing-card-container"]} ${props.typePlan.plan === "standard plan" ? classes["standard-plan"] : ""} ${
```

Comment:
Instead of having logic for classes here I think its better to have it outside as variable. For example:
`const planClass = ${props.typePlan.plan === "standard plan" ? classes["standard-plan"] : props.typePlan.plan === "pro plan" ? classes["pro-plan"] : ""`